packages/fast-components-styles-msft/src/patterns/input-field.ts

```diff
@@ -63,7 +61,26 @@ export function inputFieldStyles(
             ...applyDisabledState(),
         },
         "&::placeholder": {
-            color: neutralForegroundHint,
+            color: neutralForegroundHint(neutralFillInputRest),
+        },
+    };
+}
+
+export function filledInputFieldStyles(): CSSRules<{}> {
+    return {
+        ...inputFieldStyles(),
+        background: neutralFillRest,
+        border: format("{0} solid transparent", toPx(outlineWidth)),
+        "&:hover:enabled": {
+            background: neutralFillHover,
+            borderColor: "transparent",
+        },
+        "&:active:enabled": {
+            background: neutralFillActive,
+            borderColor: "transparent",
+        },
```

Comment:
it saves a small amount of space to do `"&:hover:enabled, &:active:enabled"` for your borderColor property.

Reply:
Turns out this broke something with rule specificity and hover started taking over focus. Leaving as is.